docs/PostgreSQL.md

```diff
@@ -111,4 +111,34 @@ Eclair stores the latest database settings in the `${data-dir}/last_jdbcurl` fil
 
 The node operator can force Eclair to accept new database 
 connection settings by removing the `last_jdbcurl` file. 
-    
\ No newline at end of file
+
+### Migrating from Sqlite to Postgres
```

Comment:
@sstone @t-bast wanna try following this migration guide and report any issues? 🙏